lib/cisco_node_utils/bgp.rb

```diff
@@ -152,6 +158,17 @@ def bestpath_med_non_deterministic
       config_get('bgp', 'bestpath_med_non_deterministic', @get_args)
     end
 
+    # nsr Setter
+    def nsr=(enable)
+      if platform == :nexus || @vrf != 'default'
+        fail Cisco::UnsupportedError.new('bgp', 'nsr', 'set', 'nsr is not ' \
```

Comment:
Could you make the error text a little more descriptive?  Something like this:
"nsr is not configurable ' \
'on a per-VRF basis on Nexus')

(I used other msgs in bgp.rb as an example)